2.4/2.4.1/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+<script>
+	let adnim;
+	let name = 'Джон';
+	adnim = name;
+	alert(adnim);
```

Comment:
Please add empty lines between blocks like this:
![image](https://github.com/tacya9/JS-1-2/assets/115877101/c42d8253-d0c6-4477-9589-3a00c2ab3634)